src/chrome/content/rules/deutschepost.xml

```diff
@@ -1,5 +1,5 @@
-<ruleset name="deutschepost">
+<ruleset name="deutschepost" default_off="FF37+ doesn't connect">
```

Comment:
Please add a link to the issue in a comment above the ruleset, so future editors can find out more details.